content/en/_index.md

```diff
@@ -946,7 +946,20 @@ Follow the [Google Java Style Guide](https://google.github.io/styleguide/javagui
 Tooling available at [google/google-java-format](https://github.com/google/google-java-format).
 {{% /tab %}}
 
+{{% tab header="Go" text=true %}}
+Follow the [Google Go Style Guide](https://google.github.io/styleguide/go/) and [Effective Go](https://go.dev/doc/effective_go).
+
+The official linters for go are [gofmt](https://pkg.go.dev/cmd/gofmt), [go vet](https://pkg.go.dev/cmd/vet), and [go lint](https://github.com/golang/lint).
+{{% /tab %}}
+
+{{% tab header="Python" text=true %}}
+All Python samples should follow the best practices defined in the [PEP 8](https://www.python.org/dev/peps/pep-0008/) style guide and the [Google Python Style Guide](http://google.github.io/styleguide/pyguide.html).
```

Comment:
Remove the Google Python Style Guide. It conflicts with our standard style (such as indentation amount). For public samples, we follow the public Python PEP guide.